tgui/packages/tgui/interfaces/RCD.js

```diff
@@ -98,6 +98,18 @@ const AirlockSettings = (props, context) => {
           />
         </LabeledList.Item>
       </LabeledList>
+      <box hidden={!airlock_glass}>
+        <LabeledList>
+          <LabeledList.Item color="silver" label="Electrochromic">
+            <Button
+              icon={electrochromic ? 'toggle-on' : 'toggle-off'}
+              content={electrochromic ? "On" : "Off"}
+              selected={electrochromic}
+              onClick={() => act('electrochromic')}
+            />
+          </LabeledList.Item>
+        </LabeledList>
+      </box>
```

Comment:
```suggestion
        {airlock_glass &&
          <LabeledList.Item color="silver" label="Electrochromic">
            <Button
              icon={electrochromic ? 'toggle-on' : 'toggle-off'}
              content={electrochromic ? "On" : "Off"}
              selected={electrochromic}
              onClick={() => act('electrochromic')}
            />
          </LabeledList.Item>
          }
      </LabeledList>
```
this will be a better way to do what you're trying to do here. You'll need to recompile TGUI and fix the formatting on my suggestion since I'm doing this through the brower.

Reply:
Putting it into the same list as the airlock name will cause it and its button to shift when electrochromic is displayed or hidden. I'll put the visibility suggestion, but I'd rather leave the electro section be its own list.
![RCDButtons2](https://user-images.githubusercontent.com/80771500/187087316-a1ae9864-3ace-4b3f-a106-7729a94d119d.PNG)
![RCDButtons](https://user-images.githubusercontent.com/80771500/187087281-07c429c2-17cd-4616-8e10-8b8809d68175.PNG)

Reply:
👍 
Alright that's fine with me